src/java.base/share/conf/security/java.security

```diff
@@ -185,7 +185,7 @@ RestrictedSecurity.NSS.140-2.securerandom.algorithm = PKCS11
 RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.desc.name = OpenJCEPlusFIPS Cryptographic Module FIPS 140-3
 RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.desc.default = false
 RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.desc.fips = true
-RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.desc.hash = SHA256:e71c49d65fd291efe75993ccbe6999e6cfb26bf9ef3e8424cb086c7e2a225ce6
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3.desc.hash = SHA256:95b4adb82ccebe4ac3800bf41664b646b0e4dd9312a946b70456d83ff4284326
```

Comment:
My testing suggests the hash should be `dd19c8f8f2578cf400c11b5c7d003684cba5fc4999ac5c55d2a73099f70f9582`.

Reply:
Right, the order matters. Updated the hash value.